bin/cli.sh

```diff
@@ -12,12 +12,22 @@ VERSION=$(cat VERSION)
 DEFAULT_IMAGE=faasm/cli:${VERSION}
 INNER_SHELL=${SHELL:-"/bin/bash"}
 export CLI_IMAGE=${1:-${DEFAULT_IMAGE}}
+export COMPOSE_PROJECT_NAME="faasm-dev"
 echo "Running Faasm CLI (${CLI_IMAGE})"
 
-# Run shell in CLI container
+## Run shell in CLI container
+# If service not running, bring it up first.
+# Then attach to it (note we `up` containers in dettached mode).
+if [[ -z $(docker ps -q --filter name="faasm-dev_cli_1") ]]; then
```

Comment:
Do we need this `if` line? Won't `docker-compose up` implicitly do this (i.e. it will ensure that the stuff is already running)?

Reply:
The `if` line is not actually necessary. However, to ensure that containers are not recreated as we re-issue a `docker-compose up` command, we make sure we launch them using the `--no-recreate` flag.